EmaPajic/assignment3/assignment3.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: EmaPajic
+"""
+
+from dictionary import Dictionary
+
```

Comment:
Optional: if you created a class including all the methods for the word search you could precompute (or store) in the __init__() the size of the grid and not have to pass it around all the time.